Tools/build/compute-changes.py

```diff
@@ -28,13 +28,17 @@
 })
 SUFFIXES_C_OR_CPP = frozenset({".c", ".h", ".cpp"})
 SUFFIXES_DOCUMENTATION = frozenset({".rst", ".md"})
```

Comment:
Keep sorted, and also rename ("config" is too generic, and the files are also used on macOS, which isn't Linux)
```suggestion
UNIX_BUILD_FILE_NAMES = frozenset({
    "aclocal.m4",
    "config.guess",
    "config.sub",
    "configure",
    "configure.ac",
    "install-sh",
    "Makefile.pre.in",
})

SUFFIXES_C_OR_CPP = frozenset({".c", ".h", ".cpp"})
SUFFIXES_DOCUMENTATION = frozenset({".rst", ".md"})
```